canvas_modules/common-canvas/src/tooltip/tooltip.scss

```diff
@@ -30,7 +30,7 @@
 	line-height: 1.2;
 	text-align: left;
 	z-index: 10000; // Modal layout has z-index 9000. Show tooltip on top of modal.
-	pointer-events: none;
+	// pointer-events: none;
```

Comment:
If this is not needed it should be deleted not commented out.  (Note: There shouldn't be any commented out code merged into main unless there is a comment to indicate why the code is commented out.)